docs/api.md

```diff
@@ -107,11 +107,12 @@ In the future we expect that there will be more API groups, all at REST path `/a
 using `apiVersion: $API_GROUP/$VERSION`.  We expect that there will be a way for (third parties to
 create their own API groups](design/extending-api.md), and to avoid naming collisions.
 
-## Enabling the extensions group
+## Enabling resources in the extensions group
 
-Enable `extensions/v1beta1` objects by adding the following flags to your API server:
-
-  - `--runtime-config=extensions/v1beta1=true`
+Jobs, Ingress and HorizontalPodAutoscalers are enabled by default.
+Other extensions resources can be enabled by setting runtime-config on
+apiserver. For ex: to enable deployments, set
+`--runtime-config=extensions/v1beta1/deployments=true`
```

Comment:
Yes, starting the server and seeing what is served is the only way I can think of.